STYLE.md

```diff
@@ -0,0 +1,227 @@
+# Purpose
```

Comment:
Well, I don't know how you would want to do it. A simple way to get rid of `STYLE.MD` is to link directly to the style guide in the `README.MD` or in the `CONTRIBUTING.MD`.

So, sure a central repository would work if you don't want to do webhosting or whatever.